apps/communications/contacts/js/bootstrap.js

```diff
@@ -12,6 +12,7 @@
   function loadScripts() {
     var dependencies = [
      '/contacts/services/contacts.js',
+     '/contacts/js/utilities/tagSelector.js',
```

Comment:
Don't load the tag selector on the critical path of the app, won't be needed on startup time.
A good place to lazy load this app will be at the moment that we click on the tags, in `onGoToSelectTag` seems to be a good candidate.